crt_portal/cts_forms/models.py

```diff
@@ -365,6 +365,10 @@ class ReportAttachment(models.Model):
     report = models.ForeignKey(Report, on_delete=models.CASCADE, related_name='attachments')
     created_date = models.DateTimeField(auto_now_add=True)
 
+    @property
+    def download_url(self):
+        return self.file.name
+
```

Comment:
Good opportunity here to should drop this snippet and replace with a [get_absolute_url](https://docs.djangoproject.com/en/2.2/ref/models/instances/) method. 

It's great to use `get_absolute_url` or the [url templatetag](https://docs.djangoproject.com/en/2.2/ref/templates/builtins/#url) instead of hard-coding.

Reply:
we applied get_absolute_url method. Thanks for the tips.